apps/site/assets/css/_location-card.scss

```diff
@@ -21,65 +18,54 @@
 .c-location-card {
   background-color: $white;
   border: 1px solid $gray-lightest;
-  border-radius: $space-1 * 3;
-  color: $gray-dark;
-  display: inline-block;
-  margin: $base-spacing / 2;
   padding: $base-spacing;
-  width: 100%;
-}
 
-.c-location-card--flex {
-  @include media-breakpoint-only(xs) {
-    flex: 1 1 100%;
-    margin: .25rem;
-    padding: .5rem 1rem;
+  &__place {
+    font-size: 1.5rem;
+    font-weight: $headings-font-weight;
+    line-height: 1.9rem;
+    @include media-breakpoint-down(xs) {
+      font-size: 1.15rem;
+    }
+    width: 67%;
+    overflow: hidden;
```

Comment:
I'm not sure if these are still needed since it's in the `justify-content: space-between` area?

Reply:
Ohh I think I see, it's helping make enough room to keep the distance visible? It might make more sense to specify the width for the companion `.c-location-card__distance` instead (or maybe a `white-space: nowrap` there, with a `hyphens: auto` here so long place names won't push the distance out of the container? hmmm)

Reply:
Yessss, ham-jamming it like I did was not the best way of doing that 😅 I adjusted it to use `text-overflow: ellipsis`, which isn't as great as hyphens, but I couldn't get hyphenation to work in a reliable way